demo/rube_testbed/loadrube.js

```diff
@@ -37,7 +37,11 @@ function loadBodyFromRUBE(bodyJso, world) {
     else
         bd.set_position(new Box2D.b2Vec2(0,0));
 
-    bd.set_gravityScale(bodyJso.gravityScale || 1);
+    if (bodyJso['gravityScale'] != undefined) {
```

Comment:
It's check not a string. It check if "gravityScale" property of json object "bodyJso" is defined. If defined then set it. Because in old version if "bodyJso.gravityScale" was 0 due not strict equality of this expression "bodyJso.gravityScale || 1" gravity became 1, instead of 0